src/RabbitMQContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using RabbitMQ.Client;
+
+namespace Microsoft.Azure.WebJobs.Extensions
+{
+    public class RabbitMQContext
+    {
+        public string Hostname { get; set; }
+
+        public string QueueName { get; set; }
+
+        public string Message { get; set; }

```

Comment:
Same here delete Message

Reply:
I thought I needed to include Message in Context/Attribute/Options so that the message could be passed into the output binding as a parameter and sent to the queue. Is there some other way to do this?

Reply:
Message should not be part of Attribute/Options.  Lets follow up with @brettsam on how to enable message for autoresolve